tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_bug_1116566.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+from marionette.by import By
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBug1116566(GaiaTestCase):
+    _menu_button_locator = (By.CSS_SELECTOR, '.msg-folder-list-btn')
+
+    def setUp(self):
+        try:
+            self.testvars['email']['IMAP']
+        except KeyError:
+            raise SkipTest('account details not present in test variables')
+
+        GaiaTestCase.setUp(self)
+        self.connect_to_local_area_network()
+
+        # Open email app
+        self.email = Email(self.marionette)
+        self.email.launch()
+
+    def test_bug_1116566(self):
+        """ https://bugzilla.mozilla.org/show_bug.cgi?id=1116566 """
+        # setup email account
+        self.email.setup_IMAP_email(self.testvars['email']['IMAP'])
+
+        # check account has emails
+        self.email.wait_for_emails_to_sync()
+        self.assertGreater(len(self.email.mails), 0)
+
+        self.assertEqual(len(self.marionette.find_elements(*self._menu_button_locator)), 1, 'Should have only 1 menu-button')
+
+        # Pull up the cards view
+        self.device.hold_home_button()
+        cards_view = CardsView(self.marionette)
+        cards_view.wait_for_cards_view()
+        cards_view.wait_for_card_ready('email')
```

Comment:
I think using `self.email.name` would be more appropriate. It'd help if we change the name of the app one day (this happened with the homescreen last summer)